source/code/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -77,29 +77,10 @@ namespace PlayFab
 
     bool CallRequestContainer::HandleInvalidSettings()
```

Comment:
should this be a void return now that it always returns true?

and perhaps the name could be updated to something that more explicitly signifies exception throwing like `ValidateSettings` or `ThrowIfSettingsInvalid`